src/_data/team.yml

```diff
@@ -721,3 +721,21 @@
   linkedin: john-rihana-861b27a6
   email: john.rihana@codurance.com
   office: london
+
+- id: domingo_espinosa
+  name: Domingo Espinosa
+  image: /assets/custom/img/team/domingo_espinosa.jpg
+  role: Apprentice
+  fullDescription:
+      <p>Domingo Espinosa is a software developer with 10 years of experience in different companies and sectors.</p>
+      <p>He is a Java specialist, always looking for continues improvement. He is a great team maker with a serene and warm personality.</p>
+      <p>During his University years, he was very involved in the Junior Enterprise of Gran Canaria, GCC, occupying different roles and positions, including president.</p>
+      <p>When he finished his degree, together with a partner, he developed software for companies in Gran Canaria.</p>
+      <p>Looking to expand his professional expertise, he moved to Madrid and a few years later to Barcelona, where he worked in different companies of national relevance.</p>
+      <p>In the quest of improving, after some meetups, he discovers a better approach to coding and programming. This is when gets into the Software Craftsmanship’s world.Since then, he has implemented this path.</p>
+      <p>For him, being part of the Codurance apprentice program, is fulfilling  the dream of being a Craftsman. In the future he would like be able to guide others and transmit his passion and expertise.</p>
+      <p>In his spare time, Domingo is dedicated to training for triathlons and open waters swimming. He likes to spend time with his family.</p>
```

Comment:
I think it would be `In his spare time, Domingo trains for triathlons and swims in open waters.`